packages/components/src/higher-order/with-filters/index.js

```diff
@@ -28,6 +28,7 @@ export default function withFilters( hookName ) {
 			/** @inheritdoc */
 			constructor( props ) {
 				super( props );
+				this.ComponentProps = applyFilters( hookName + 'Props', props );
```

Comment:
I think this should be called also inside `throttledForceUpdate`

Reply:
good point - thanks for the feedback - addressed in https://github.com/WordPress/gutenberg/pull/10195/commits/08f27c335895dd6150f4bbd91b09e8e904617f15. If you think this might be something we would adopt, I can take a pass at adding some tests.